cmake/Modules/FindCUDNN.cmake

```diff
@@ -23,7 +23,7 @@ find_path(CUDNN_INCLUDE cudnn.h
   PATHS ${CUDNN_ROOT} $ENV{CUDNN_ROOT}
   DOC "Path to cuDNN include directory." )
 
-find_library(CUDNN_LIBRARY NAMES libcudnn.so cudnn.lib # libcudnn_static.a
+find_library(CUDNN_LIBRARY NAMES libcudnn.so cudnn.lib cudnn
```

Comment:
Can you explain why this change is needed?

Reply:
Added it while testing. Forgot to remove it. Will do